tests/conftest.py

```diff
@@ -15,9 +17,45 @@ def valid_product_design_space_data():
         schema_id='6c16d694-d015-42a7-b462-8ef299473c9a',
         id=str(uuid.uuid4()),
         config=dict(
-            type='Univariate',
+            type='ProductDesignSpace',
             name='my design space',
             description='does some things',
+            subspaces=[
+                dict(
+                    module_type='DESIGN_SPACE',
+                    status='READY',
+                    id=str(uuid.uuid4()),
+                    archived=False,
+                    name='first subspace',
+                    instance=dict(
```

Comment:
The subspace modules come back inside an `instance` field instead of a `config` field. I don't know why.